packages/atoms/src/components/kiwi-dropdown/kiwi-dropdown.spec.ts

```diff
@@ -13,6 +14,44 @@ const newKiwiDropdown = (): Promise<SpecPage> =>
   });
 
 describe('kiwi-dropdown', () => {
+  it.each`
+    left    | right   | expected
+    ${50}   | ${350}  | ${'0px'}
+    ${1700} | ${2000} | ${'-80px'}
+    ${100}  | ${3000} | ${'-100px'}
+  `(
+    'translates the dropdown { left: $left, right: $right } horizontally by $expected for FullHD screen',
+    async ({ left, right, expected }) => {
+      const dropdown = await newKiwiDropdown();
+
+      if (dropdown.win.visualViewport === undefined) {
+        Object.defineProperty(dropdown.win, 'visualViewport', {
+          value: { width: 1920 },
+        });
+      }
```

Comment:
```suggestion
      Object.defineProperty(dropdown.win, 'visualViewport', {
        value: { width: 1920 },
        writable: true,
      });
```
Ich habe nochmal rumgespielt und herausgefunden, dass `writable: true` zulässt, die Property zu überschreiben.
`jest.spyOn(dropdown.win, 'visualViewport', 'get').mockReturnValue({…})` geht leider nicht, weil `visualViewport` anfangs `undefined` ist.

Reply:
Okay, habe ich so umgebaut. Danke für den Vorschlag!